.travis.yml

```diff
@@ -18,14 +19,21 @@ gemfile:
   - gemfiles/rails_4_2.gemfile
 matrix:
   exclude:
-    - rvm: 2.6.5
+    - rvm: 2.7.1
       gemfile: gemfiles/rails_4_2.gemfile
-    - rvm: 2.4.9
+    - rvm: 2.6.6
+      gemfile: gemfiles/rails_4_2.gemfile
+    - rvm: 2.4.10
       gemfile: gemfiles/rails_6_0.gemfile
-cache: bundler
+cache:
+  directories:
+    - /home/travis/.rvm/gems/ruby-2.7.1
```

Comment:
Caching was not working properly before. `cache: bundler` assumes that the `install` command is not being overridden — if that is the case then Bundler will use `vendor/bundle` as the installation path. However, since we _are_ overriding `install` and we are not providing a `--path` to `bundle install`, Travis will install gems in a global location, which happens to be under `/home/travis/.rvm/gems/ruby-<version>`. Here I'm listing out all of the possible cache paths because it will change depending on which Ruby version the build is running under.